docs/source/general.md

```diff
@@ -0,0 +1,160 @@
+conda.enable = true
+process {
+    // Use Conda environment files
+    withName:scSplit {
+        conda = './conda/scsplit.yml'
+    }
+    // Use Conda package names
+    withName:cellSNP {
+        conda = 'bioconda::cellsnp-lite'
+    }
+    // Use existing Conda environments
+    withName:summary {
+        conda = '/path/to/an/existing/env/directory'
+    }
+}
+
+```
+
+### Containers:
+
+Nextflow also supports a variety of container runtimes, e.g. Docker. To specify a different Docker image for each process:
+
+```
+process {
+    withName:foo {
+        container = 'image_name_1'
+    }
+    withName:bar {
+        container = 'image_name_2'
+    }
+}
+// do not forget to enable docker
+
+docker.enabled = true
+
+```
+
+### Executor and resource specifications:
+
+- The pipeline can be run either locally or on an HPC. You can set the executor by running the pipeline with `-profile standard` or `-profile cluster`. Of course, you can add other profiles if you want.
+- Feel free to add other configurations, e.g. the number of CPUS, the memory allocation, etc. If you are new to Nextflow framework, please visit the [Nextlfow page](https://www.nextflow.io/docs/latest/config.html#).
+- As default, the pipeline is run locally with the standard profile, where all processes annotated with the big_mem label are assigned 4 cpus and 16 Gb of memory.
+
+```
+profiles{
+    standard {
+        process {
+            executor = 'local'
+            withLabel: big_mem {
+                cpus = 4
+                memory = 16.GB
+            }
+            withLabel: small_mem {
+                cpus = 2
+                memory = 8.GB
+            }
+        }
+
+    }
+
+    cluster {
+        process {
+            executor = 'slurm'
+             // queue = ...
+            withLabel: big_mem {
+                cpus = 32
+                memory = 64.GB
+            }
+            withLabel: small_mem {
+                cpus = 16
+                memory = 32.GB
+            }
+        }
+    }
+}
+
+```
+
+## **Advanced usecases**
+
+### **Running on multiple samples**
+
+The pipeline is able to run on multiple samples. In this scenario, the shared parameters for input data are retrieved from a sample sheet using `params.multi_sample`, which is set to None by default. Along with the input data, the sample sheet should contain an additional column for unique sample IDs assigned to each sample. The remaining parameters for each process are specified in the nextflow.config file, just like when demultiplexing a single sample. However, there is a distinction between running on a single sample and running on multiple samples. When processing multiple samples, the pipeline only permits a single value for each process parameter, whereas in the case of a single sample, multiple values separated by commas are allowed. The sample sheet (example file see the Resources section below) should have e.g. following columns depending on the methods you want to run:
```

Comment:
Please make more line breaks after the sentences. It makes it easier to review...